server/repository/src/db_diesel/requisition/mod.rs

```diff
@@ -145,28 +145,24 @@ impl RequisitionFilter {
         self.their_reference = Some(filter);
         self
     }
+
+    pub fn new_match_id(id: &str) -> RequisitionFilter {
```

Comment:
I'm not sure about the naming of this.
`new_match_id` doesn't feel quite right. I like the concept though.

How about `by_id` instead? It doesn't feel quite right either but feels better to me.

Example usage would change from 

`repo.query_one(RequisitionFilter::new_match_id(id))`
to 
`repo.query_one(RequisitionFilter::by_id(id))`

Reply:
Agree, thanks, also found it to be not feel quite right: c1003bf9980f6c3a44daf17a6bed6a8f8908dfca

Reply:
Sorry I also should have done a commit here